Makefile

```
@@ -13,17 +13,17 @@
 
 IMAGE_NAME := fluent/fluentd
 X86_IMAGES := \
-	v1.16/alpine:v1.16.3-1.0,v1.16-1,edge \
-	v1.16/debian:v1.16.3-debian-amd64-1.0,v1.16-debian-amd64-1,edge-debian-amd64
+	v1.16/alpine:v1.16.3-1.1,v1.16-1,edge \
+	v1.16/debian:v1.16.3-debian-amd64-1.1,v1.16-debian-amd64-1,edge-debian-amd64
```

Although we don't yet determine the version rule for this upgrade pattern (upgrading Ruby without upgrading fluentd), it might be better to increment major revision instead of minor:
```suggestion
	v1.16/alpine:v1.16.3-2.0,v1.16-2,edge \
	v1.16/debian:v1.16.3-debian-amd64-2.0,v1.16-debian-amd64-2,edge-debian-amd64
```
Please note that current build system might not suppose such versioning number, we need to check it if we decide to apply such versioning rule.

---

Thanks!
I've fixed it.